exercises/I-array-properties/exercise.js

```diff
@@ -5,8 +5,13 @@
 
 */
 
-function isEmpty(arr) {
-  return; // complete this statement
+function isEmpty(arr) { // complete this statement
+  if (arr.length === 0){
```

Comment:
Could replace the if-else with a predicate similar to the function below
function isNegative(number) {
  return number < 0;
}